crates/pera-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -3340,6 +3388,11 @@ impl AuthorityPerEpochStore {
         let tracking_id = transaction.get_tracking_id();
 
         match &transaction {
+            SequencedConsensusTransactionKind::External(ConsensusTransaction {
+                kind:
+                    ConsensusTransactionKind::SignatureMPCOutput(statements, session_id, sender_address),
+                ..
+            }) => Ok(ConsensusCertificateResult::ConsensusMessage),
             SequencedConsensusTransactionKind::External(ConsensusTransaction {
```

Comment:
@ItayLevyOfficial @yaelAbergel

Reply:
@ycscaly Re _This must only be reached in case 2/3 of the validators agreed on this output._ - this is not true. If a validator sends an output TX, all the validators always receive and handle it. Before replacing it with the system TX tho, there's a verification this output is valid, and then, only if more then 2/3 of the voting power decides to create the system TX, it's being stored on chain. If this output is not valid, the sending validator is being marked as malicious.